### PR TITLE
Enrich telemetry data with host properties

### PR DESCRIPTION
**Pull request by kke**
Signed-off-by: Kimmo Lehto <klehto@mirantis.com>

* Adds host properties (cpu count & architecture, os name & version) to the telemetry context info
* Moves the k0s version from event properties into the intended place: analytics.Context.App.Version

### CODE REVIEW

File: pkg/telemetry/telemetry.go [hunk 1]
```diff
@@ -140,14 +141,37 @@ func (c Component) sendTelemetry() {
...
+	}
+
+	hostData.App.Version = c.Version
+	hostData.App.Name = "k0s"
+	hostData.App.Namespace = "k0s"
+	hostData.Extra["cpuArch"] = runtime.GOARCH
+
+	if runtime.GOOS == "linux" {
+		var si sysinfo.SysInfo
+		si.GetSysInfo()
```
Comment (kke) [inline pkg/telemetry/telemetry.go:156]: Not sure if this gets heavy, it does all kinds of data retrieval from who knows where.

Comment (jnummelin) [reply pkg/telemetry/telemetry.go:156]: the telemetry is collected anyway once per 10mins by default, so IMO not an issue

**Comment by kke** [comment]
I guess this can be reviewed.

**Review by soider — APPROVED** [review]
looks good

**Pull request merged**